src/llmq/instantsend.cpp

```diff
@@ -1737,7 +1737,7 @@ bool CInstantSendManager::IsInstantSendMempoolSigningEnabled() const
 
 bool CInstantSendManager::RejectConflictingBlocks() const
 {
-    if (!masternodeSync->IsBlockchainSynced()) {
+    if (masternodeSync == nullptr || !masternodeSync->IsBlockchainSynced()) {
```

Comment:
Is the whole expression evaluated before execution or is only the first valid expression evaluated?

Reply:
it'll short circuit, so if the first part is true, then we will return false